.github/workflows/replace_readme.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v1
```

Comment:
seems we can change this to v4 can you also help to change it?

Reply:
ok, I've changed it.